diff --git a/node_modules/@react-navigation/elements/src/useFrameSize.tsx b/node_modules/@react-navigation/elements/src/useFrameSize.tsx
index d692dd8..08df23a 100644
--- a/node_modules/@react-navigation/elements/src/useFrameSize.tsx
+++ b/node_modules/@react-navigation/elements/src/useFrameSize.tsx
@@ -14,10 +14,10 @@ import { useSyncExternalStoreWithSelector } from 'use-sync-external-store/with-s
 
 // Load with require to avoid error from webpack due to missing export in older versions
 // eslint-disable-next-line import-x/no-commonjs
-const SafeAreaListener = require('react-native-safe-area-context')
+/*const SafeAreaListener = require('react-native-safe-area-context')
   .SafeAreaListener as
   | typeof import('react-native-safe-area-context').SafeAreaListener
-  | undefined;
+  | undefined; */
 
 type Frame = {
   width: number;
